get.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Get Plasma Mobile
+parmalink: /get
+layout: default
+---
+ 
+Distributions offering Plasma Mobile
+====================================
+
+**Mobile**
+----------
+
+Neon based reference rootfs (recommended)
+----------------------------------------
+
+![](/img/neon.svg){:width="100"}
+
+"Official" image, based on KDE Neon. Targetted at newer devices (devices with 1GB+ RAM).
+KDE Neon itself is based upon Ubuntu 16.04 (xenial), (not Ubuntu Touch).
```

Comment:
I don't think the

> (not Ubuntu Touch)

is really necessary

Reply:
Yeah, I'm not sure there either, but sometimes users where confused and did not understand if Plasma Mobile uses Ubuntu or Ubuntu Touch as base.